source/Irrlicht/COpenGLFrameBuffer.cpp

```diff
@@ -180,7 +180,7 @@ bool COpenGLFrameBuffer::attach(E_FBO_ATTACHMENT_POINT attachmenPoint, core::sma
     }
     if (enabledBufferCnt)
         enabledBufferCnt += 1-EFAP_COLOR_ATTACHMENT0;
-    COpenGLExtensionHandler::extGlNamedFramebufferDrawBuffers(frameBuffer, enabledBufferCnt, drawBuffers);
+    COpenGLFunctionTable::extGlNamedFramebufferDrawBuffers(frameBuffer, enabledBufferCnt, drawBuffers);

```

Comment:
nope the methods of the function table cannot be static, you need `getThreadContext_helper` then get the function table from the `SAuxContext` and then call the non-static method

Reply:
confer with https://github.com/buildaworldnet/IrrlichtBAW/blob/6b7aeccf2d8030799992b651fcd5546d10f79e48/source/Irrlicht/COpenGLDriver.h#L586